src/components/scores/RoomDialogLink.tsx

```diff
@@ -30,9 +36,10 @@ export default function RoomDialogLink({ id, label, eventId, databaseId, meetId,
             data-match-id={matchId}
             data-room-id={roomId}
             onClick={e => window.openRoomDialog(e)}
-            onMouseOver={e => (e.currentTarget.style.textDecoration = "underline")}
-            onMouseOut={e => (e.currentTarget.style.textDecoration = "none")}
+            onMouseOver={e => (e.currentTarget.style.fontWeight = "bold", e.currentTarget.style.color = "var(--accent-focus)", e.currentTarget.style.textDecoration = "underline", e.currentTarget.style.textUnderlinePosition = "under") }
```

Comment:
This seems really weird. Why is this not being done with a :hover style in the `custom.css`?